download_and_create_reference_datasets/v03/write_clinvar_ht.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+import argparse
+import os
+
+import hail as hl
+
+from hail_scripts.reference_data.constants import GCS_PREFIX
+from hail_scripts.utils.clinvar import (
+    download_and_import_latest_clinvar_vcf,
+    CLINVAR_GOLD_STARS_LOOKUP,
+)
+from hail_scripts.utils.hail_utils import write_ht
+
+CLINVAR_HT_PATH = os.path.join(GCS_PREFIX, 'clinvar/clinvar.GRCh{genome_version}.{timestamp}.ht')
+
+def run(environment: str):
+    for genome_version in ['37', '38']:
+        mt = download_and_import_latest_clinvar_vcf(genome_version)
+        timestamp = hl.eval(mt.version)
+        ht = mt.rows()
+        ht = ht.annotate(
+            gold_stars=CLINVAR_GOLD_STARS_LOOKUP.get(hl.delimit(ht.info.CLNREVSTAT))
+        )
+        ht.describe()
+        ht = ht.transmute(info=ht.info.select('ALLELEID', 'CLNSIG')).select('info', 'gold_stars')
+        ht = ht.repartition(100)
+        destination_path = CLINVAR_HT_PATH.format(
+            environment=environment,
+            genome_version=genome_version,
+            timestamp=timestamp,
+        )
+        print(f'Uploading ht from {hl.eval(ht.sourceFilePath)} to {destination_path}')
+        write_ht(ht, destination_path)
+        os.remove(ht.sourceFilePath)
```

Comment:
Should we also delete the downloaded VCF file?

Reply:
`sourceFilePath` is the VCF file here.  the `ht` is only in memory.

Reply:
well... until we write to `gcs`.

Reply:
if we are running this on dataproc do we need to clean up the file system?

Reply:
Honestly the cleanest answer here is to do all this work within the `with tempfile.NamedTemporaryFile(...)` context manager which will manage cleanup itself.  We've got some work within the context manager and some outside which is why this cleanup line kinda seems out of place.